components/prombench/manifests/benchmark/3b_prometheus-test.yaml

```diff
@@ -0,0 +1,306 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-test-restarter-scripts
+  namespace: prombench-{{ .PR_NUMBER }}
+data:
+  restarter.sh: |
+    #!/bin/sh
+    echo "Restarting prometheus"
+    kill -SIGHUP $(pidof start.sh)
+  killer.sh: |
+    #!/bin/sh
+    echo "Killing prometheus"
+    kill -9 $(pidof prometheus)
+  fetchtini.sh: |
```

Comment:
we can move the tini download part to the prometheus-builder image
then run prometheus-builder for both deployments

- the release deployment should only copy tini to the required directory
- the pr deployment will work same + copy tini to the required directory

this way we won't need to run this script from k8s and reduce the number of
initContainers by 1 for pr deployment